airbyte-webapp/.stylelintrc

```
@@ -6,7 +6,7 @@
     "stylelint-config-prettier-scss"
   ],
   "rules": {
-    "selector-class-pattern": "^[a-z][a-zA-Z0-9]+$",
+    "selector-class-pattern": "^[a-z][a-zA-Z0-9_/-]+$",
```

```suggestion
    "selector-class-pattern": "^[a-z][a-zA-Z0-9_-]+$",
```
I double checked with https://regex101.com/r/RRVDsJ/1 that actually the escaping here wouldn't be needed.